server/public/model/property_field.go

```diff
@@ -166,11 +166,11 @@ func (p PropertyFieldSearchCursor) IsValid() error {
 	}
 
 	if p.CreateAt <= 0 {
-		return errors.New("create at cannot be negative or zero")
+		return fmt.Errorf("create_at value %d cannot be negative or zero", p.CreateAt)
```

Comment:
Should we also use `%q` for integers to be consistent with the string formatting?

Reply:
It's not necessary, integers cannot have spaces. %q is only meant for string values which can have spaces.